serl_robot_infra/franka_env/envs/franka_env.py

```diff
@@ -223,10 +226,15 @@ def compute_reward(self, obs) -> bool:
         current_pose = np.hstack([current_pose[:3], euler_angles])
         delta = np.abs(current_pose - self._TARGET_POSE)
         if np.all(delta < self._REWARD_THRESHOLD):
-            return True
+            reward = 1
         else:
             # print(f'Goal not reached, the difference is {delta}, the desired threshold is {_REWARD_THRESHOLD}')
-            return False
+            reward = 0
+
+        if gripper_action_effective:
```

Comment:
we should make a flag if enabling this reward or not, otherwise it'll apply this thing across the board even you don't need a gripper

Reply:
I've set the penalty value to 0 in the config for environments that don't operate the gripper

Reply:
I think we still should set a flag, it's less error-prone.  for environments don't operate the gripper, we shouldn't be compute gripper_action_effective at all.